spec/lib/rmt/cli/main_spec.rb

```diff
@@ -115,7 +101,7 @@
           end
         end
 
-        context 'Mysql2::Error with cannot connect error' do
```

Comment:
What's wrong with contexts?

Reply:
Well, one is a context, the other describes a thing :nerd_face:  Technically they are almost always interchangeable but readability suffers if you use them wrong.

Reply:
Oh, OK. It actually doesn't make sense when you try and read it with contexts. :+1: